tests/test_data/test_datasets/test_panoptic_dataset.py

```diff
@@ -0,0 +1,70 @@
+import os.path as osp
```

Comment:
The unit test coverage is not enough, because the above shows that a lot of code is not covered.